native_client/java/Makefile

```diff
@@ -1,31 +1,178 @@
-.PHONY: clean apk-clean
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
You cant over write our makefile

Reply:
You're correct yeah, that one should not be overriden, because it's not used for the standalone java. Will fix.